### ISSUE TITLE
Customisable precision for networked floats

### ISSUE THREAD

**Issue by Admer456** [Admer456/ioq3-burek#28]
At the moment, AFAIK, when a float variable is networked, all 32 bits are sent (unless it's "snapped", in which case it's a 13-bit integer).

Some floats don't have to be that precise; it'd be better to let the programmer define the number of bits and the precision bits (which can be negative).

The way it'd work is:
If the precision bits are, for example, 3, then the float will be accurate down to 0.125. If it's 2, then 0.25.
So basically, if the precision bits number is positive, the sending float gets multiplied by them while the receiving float gets divided by that number.
